backend/IDE.BLL/Services/EmailService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using IDE.BLL.Interfaces;
+using SendGrid;
+using SendGrid.Helpers.Mail;
+
+namespace IDE.BLL.Services
+{
+    public class EmailService : IEmailService
+    {
+        const string currentAddress = "https://bsa-ide.azurewebsites.net"; //How to set it
```

Comment:
this should be dynamic. Or at least loaded from the config

Reply:
Move to appsettings